### PR TITLE
Add check-store-layers check to the CI and include their updates

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Adds a check in parallel with the `app-layers` check to check that the `store-layers` are updated.

image

### CODE REVIEW

File: store/opentracing_layer.go [hunk 1]
```diff
@@ -5431,6 +5449,42 @@ func (s *OpenTracingLayerRoleStore) Save(role *model.Role) (*model.Role, *model.
 	return resultVar0, resultVar1
 }
 
+func (s *OpenTracingLayerSchemeStore) CountByScope(scope string) (int64, *model.AppError) {
```
Comment (ethervoid) [inline store/opentracing_layer.go:5452]: No need to add these changes, I've made a PR for it [here](https://github.com/mattermost/mattermost-server/pull/14261)

Comment (mgdelacroix) [reply store/opentracing_layer.go]: Great! Updated the PR to remove everything but the check itself then :)

**Comment by mgdelacroix** [comment]
Marking this PR as blocked by https://github.com/mattermost/mattermost-server/pull/14261 as the `check-store-layers` will fail until that PR is merged

**Pull request merged**